db/migrate/20250828075341_add_paid_at_to_registration_payment.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+class AddPaidAtToRegistrationPayment < ActiveRecord::Migration[7.2]
+  def change
+    add_column :registration_payments, :paid_at, :datetime
+
+    reversible do |direction|
+      direction.up do
+        execute <<~SQL.squish
+          UPDATE registration_payments
+          SET paid_at = created_at
```

Comment:
What about some `WHERE paid_at IS NULL` just for good measure? I don't know how our prod server is configured, but some MySQL versions reject general `UPDATE` without `WHERE` statements outright